### ISSUE TITLE
Bad module path in v1.0.0

### ISSUE THREAD

**Issue by NathanBaulch** [muxinc/mux-go#33]
Looks like someone forgot to specify the `--git-user-id` and `--git-repo-id` flags when running openapi-generator-cli @eropple?

```
go get: github.com/muxinc/mux-go@v0.13.0 updating to
        github.com/muxinc/mux-go@v1.0.0: parsing go.mod:
        module declares its path as: github.com/GIT_USER_ID/GIT_REPO_ID
                but was required as: github.com/muxinc/mux-go
```

**Comment by NathanBaulch** [comment]
Bump @eropple

**Comment by GeneticGenesis** [comment]
Hey @NathanBaulch, thanks for raising this. We've spotted a couple of issues with the 1.0.0 release. 

For now we've backed out that release, and we'll make sure to address this before we merge back in, please continue to use 0.13.0 for now. 

Thanks.

**Comment by joncalhoun** [comment]
It is probably worth updating install instructions on the website. Just running `go get github.com/muxinc/mux-go` will result in this error. 

I believe users now need to use something more explicit, like:

```
go get github.com/muxinc/mux-go@v0.13.0
```

**Comment by obedtandadjaja** [comment]
This issue wasted 1 hour of my time. Please update the installation instruction in the README

**Comment by philcluff** [comment]
Thanks for the feedback @obedtandadjaja and @joncalhoun, we understand what's happening here, we'll resolve it in the coming day or two.

**Comment by NathanBaulch** [comment]
Pretty annoying, there's no easy way to get a bad package version removed from Google's package proxy.

**Comment by philcluff** [comment]
Yeh, I wasn't super impressed when I realised this.

**Comment by philcluff** [comment]
Hi @NathanBaulch @obedtandadjaja @joncalhoun 

This was resolved today with a release of 1.0.1, which should be picked up very soon in the caches. Unfortunately we were forced to do a major version bump even there were no breaking changes to resolve this issue.

Thanks.
